master/buildbot/steps/source/git.py

```diff
@@ -534,14 +535,16 @@ def _syncSubmodule(self, _=None):
 
     def _updateSubmodule(self, _=None):
         if self.submodules:
-            return self._dovccmd(['submodule', 'update',
-                                  '--init', '--recursive', '--force'])
+            vccmd = ['submodule', 'update', '--init', '--recursive', '--force']
```

Comment:
looks like this one is not unit tested. We would apreciate if you add a unit test for the submodule use case.